editor/README.md

```diff
@@ -3,8 +3,7 @@
 Start locally:
 
 ```bash
-pip install -r requirements.txt
-streamlit run app.py
+docker run -P -it $(docker build -q .)
```

Comment:
Note: if you add something like `--volume=~/.cache/croissant:/root/.cache/croissant`, you bind to the local filesystem. Documentation: https://docs.docker.com/engine/reference/run/#volume-shared-filesystems